doc/docs/setup.md

```diff
@@ -1,17 +1,20 @@
-# Replicache Setup
+---
+title: Integration Guide
+slug: /integration
```

Comment:
Is it intentional that the filename is setup but the slug integration?

Reply:
Not really, just an artifact of history. Fixing.